suricata/update/parsers.py

```diff
@@ -31,6 +31,11 @@
 
 default_update_yaml = config.DEFAULT_UPDATE_YAML_PATH
 
+show_advanced = False
+
+if "-s" in sys.argv or "--show-advanced" in sys.argv:
+    show_advanced = True
```

Comment:
Sorry, should have caught this earlier. Not to keen on seeing this in the global module space. I think it fits better as the first thing in the `def parser_arg()`...

Reply:
I have a small question there. Wouldn't it be necessary to set `show_advanced = True` before `global_arg` and `update_arg` are initialized? When I move it to `def parse_arg()` value for help in the options in `global_arg` are initialized with `argparse_SUPPRESS`. As a result the options are hidden when running `--show-advanced` as well.

Please correct me if I'm wrong.

Reply:
You are correct, I didn't consider they are setup in the module name space.  Please revert.

Also, can you create a new PR for this change per the guidelines instead of a force push please?  Thanks!

Reply:
Replaced with #263